.gitlab/build-deb-rpm.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+source common_build_functions.sh
+
+if [ -n "$CI_COMMIT_TAG" ] && [ -z "$DOTNET_PACKAGE_VERSION" ]; then
+  $DOTNET_PACKAGE_VERSION=${CI_COMMIT_TAG##v}
+fi
+
+curl --location --fail \
+  --output datadog-dotnet-apm.old \
+  "https://github.com/DataDog/dd-trace-dotnet/releases/download/v$DOTNET_PACKAGE_VERSION/datadog-dotnet-apm_${DOTNET_PACKAGE_VERSION}_amd64.deb"
+
+fpm --input-type deb \
+  --output-type dir \
+  --name datadog-dotnet-apm \
+  datadog-dotnet-apm.old
+
+echo -n $DOTNET_PACKAGE_VERSION > dotnet.version
```

Comment:
```suggestion
echo -n $DOTNET_PACKAGE_VERSION > auto_inject-dotnet.version
```

Reply:
I think I can do the following and not implement @chouetz comment [here](https://github.com/DataDog/dd-trace-dotnet/pull/4303/files/16477360ae4dcddf16d01718994c66ab6427d572#r1242159321). I got it right?

```suggestion
echo -n $DOTNET_PACKAGE_VERSION > $PRODUCT_NAME.version
```

Reply:
Thanks for raising the issue. Fixed in 55f3beebf2159c36e751cd513be065740f592613
The [new run](https://gitlab.ddbuild.io/DataDog/dd-trace-dotnet/-/jobs/288460516) LGTM